src/porepy/grids/mortar_grid.py

```diff
@@ -704,7 +711,11 @@ def _init_projections(
         # If the face_duplicate_ind is not given, we then assume that the primary side faces
         # already have this ordering. If the grid is created using the pp.split_grid.py
         # module this should be the case.
-        if self.num_sides() == 2 and (face_duplicate_ind is not None):
+        if (
+            self.num_sides() == 2
+            and (face_duplicate_ind is not None)
+            and self.codim < 2
```

Comment:
I presume that the combination of face_duplicate_ind and codim == 2 is not meaningful. Should we explicitly give an error in that case?

Reply:
Done